toolbox

```diff
@@ -888,8 +930,13 @@ remove_containers()
                                 ret_val=1
                                 continue
                             fi
-                            if ! has_substring "$labels" "com.redhat.component:fedora-toolbox"; then
-                                echo "$base_toolbox_command: $id is not a toolbox container" >&2
+
+                            has_substring "$labels" "com.redhat.component:fedora-toolbox"
+                            has_old_id=$?
+                            has_substring "$labels" "com.github.debarshiray.toolbox"
+                            has_new_id=$?
+                            if [ ! $has_new_id ] || [ ! $has_old_id ]; then
```

Comment:
Umm... doesn't something like this work:
```
if ! has_substring "$labels" "com.github.debarshiray.toolbox" \
   && ! has_substring "$labels" "com.redhat.component:fedora-toolbox"; then
       ...
fi
```

For example, I played around with these:
```
$ if ls /bar >/dev/null 2>/dev/null || ls /foo >/dev/null 2>/dev/null; then echo "one valid path"; fi
$ if ls /etc >/dev/null 2>/dev/null || ls /foo >/dev/null 2>/dev/null; then echo "one valid path"; fi
one valid path
$ if ls /bar >/dev/null 2>/dev/null || ls /tmp >/dev/null 2>/dev/null; then echo "one valid path"; fi
one valid path
$ if ls /etc >/dev/null 2>/dev/null || ls /tmp >/dev/null 2>/dev/null; then echo "one valid path"; fi
one valid path
```

Reply:
The tweaked version of this patch in #110 seems to work for me.